docs/source/actions.rst

```diff
@@ -102,6 +102,12 @@ Default: ``None``
 SPIDERMON_BODY_HTML_TEMPLATE
 ----------------------------
 
+String containing the location of the template for the spidermon email report.
+
+Default: ``reports/email/monitors/result.jinja``
+
+Link to the Default `Template`_.
```

Comment:
I would not link to the `master` template, because the documentation should always be specific to the version of Spidermon containing the documentation.

Maybe you could remove this line and use [`:download:`](https://stackoverflow.com/a/8622242/939364) on ``reports/email/monitors/result.jinja`` above.